Gemfile.lock

```diff
@@ -14,6 +14,9 @@ GEM
     ffi (1.15.5-x64-mingw32)
     forwardable-extended (2.6.0)
     google-protobuf (3.21.12)
+    google-protobuf (3.21.12-x64-mingw32)
+    google-protobuf (3.21.12-x86_64-darwin)
+    google-protobuf (3.21.12-x86_64-linux)
     http_parser.rb (0.8.0)
```

Comment:
@Sranjan0208 Could you please revert these?

Reply:
@Chadha93 It's done.